.changeset/config.json

```diff
@@ -9,9 +9,11 @@
   "commit": false,
   "fixed": [
     [
+      "@commercetools-applications/*",
       "@commercetools-backend/*",
       "@commercetools-frontend/*",
-      "!@commercetools-frontend/codemod"
+      "@commercetools-local/*",
+      "@commercetools-website/*"
```

Comment:
This is to keep ALL packages linked together, even the private apps (templates, playground, etc). The result is that these packages will get bumped to the same version as the others (even though they won't be released to NPM).